### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by jermanuts** [jermanuts/bad-opsec#1] [General]
If you have any links you would like to share, please open a PR or send them here.

**Comment by jermanuts** [comment]
DRAFT:

https://twitter.com/fs0c131y/status/1787852663595454807 (lockbit)

https://analyst1.com/ransomware-diaries-volume-5-unmasking-lockbit-2/

https://restoreprivacy.com/protonmail-discloses-user-data-leading-to-arrest-in-spain/ , https://web.archive.org/web/20230814144638/https://www.forbes.com/sites/thomasbrewster/2023/08/08/protonmail-fbi-search-led-to-a-suspect-threatening-a-2020-election-official/ protonmail

>SLEUTHCON 2023 - Unmasking Venom Spider: The Hunt for the Golden Chickens

https://youtu.be/u6dtdNFPp4g

**Reply by jermanuts** [reply]
Quotes:

>Remember: opsec will get you through times of no crypto better than crypto will get you through times of no opsec. ([Source](https://twitter.com/thegrugq/status/819973538723438592))

**Reply by jermanuts** [reply]
I was thinking of arranging the Readme file, make it 2 files, first is the one for new unique and new cases, and second is for old and common cases, because adding a lot if links onto 1 page is a bit overwhelming, heck I might eventually just create a website and design it properly so it have tags, search etc.

**Comment by jermanuts** [comment]
![image](https://github.com/jermanuts/bad-opsec/assets/109705802/71523f36-77d8-45dc-acc2-e5aa364ad1c9)

lmao

**Reply by verybrian** [reply]
Could you please share a link to the rest of the details about the case, evidence, etc?

**Reply by jermanuts** [reply]
It is the 2nd one in the list on this repo, Administrator of Incognito Market.

**Reply by jermanuts** [reply]
![image](https://github.com/user-attachments/assets/4b8636b6-f880-40a2-b978-a5c280a1881f)

https://www.justice.gov/usao-dc/media/1373721/dl?inline

Too many Incognito Market admin followers it seems like.

**Comment by jermanuts** [comment]
Should I replace YouTube links to privacy respecting frontends like Piped or Invidious? I would choose the latter because it has fewer problems than Piped due to most instances not proxying videos, also doesn't need JavaScript.

EDIT: but the problem is that there's always a chance that instance is down or broken by YouTube so I will have to frequently update it :/

**Comment by jermanuts** [comment]
I plan to write blog posts on some specific topics from this collection that I really enjoyed reading. The blog posts will also include suggestions on how to improve and protect your Opsec and general attacks that compromise it.
 
 Will probably start with the Incognito story and then breachforums on https://jermanuts.vercel.app/.
